docs/release-notes/january-2024.md

```diff
@@ -0,0 +1,111 @@
+---
+slug: january-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: January 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for January 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in January 2024:
+  * [<i class="fas fa-external-link fa-xs"></i>1.55.1](https://github.com/semgrep/semgrep/releases/tag/v1.55.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.55.2](https://github.com/semgrep/semgrep/releases/tag/v1.55.2)
+  * [<i class="fas fa-external-link fa-xs"></i>1.56.0](https://github.com/semgrep/semgrep/releases/tag/v1.56.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.57.0](https://github.com/semgrep/semgrep/releases/tag/v1.57.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.58.0](https://github.com/semgrep/semgrep/releases/tag/v1.58.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.59.0](https://github.com/semgrep/semgrep/releases/tag/v1.59.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+* Semgrep's Visual Studio Code extension now runs natively on Windows machines.
```

Comment:
@s-santillan This is now the only line in present tense. Do you have any thoughts on rewording? Maybe something like "Improved Semgrep's Visual Studio Code extension so that it runs natively on Windows machines" but not sure if that sounds off

Reply:
Honestly, if we were a bigger enterprise team I'm sure we'd have a strictly enforced spec by now, but the only reason I flagged 56-60 was because they were partial sentences.

There's a case for both present and past tense for release notes. I historically followed Added/Changed/Removed (I wasn't the owner before).